examples/ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -3,5 +3,6 @@
     <!-- All update modes should additionally be exluded here https://github.com/adobe/aem-core-wcm-components/blob/master/examples/ui.content/src/content/META-INF/vault/filter.xml -->
     <filter root="/conf/core-components-examples/settings/wcm/policies/core-components-examples/components/page/policy_212634895490169" mode="update"/>
     <filter root="/content/core-components-examples/library/page-authoring/aemform" mode="update"/>
+    <filter root="/content/core-components-examples/library/page-authoring/link" mode="update"/>
```

Comment:
/content/core-components-examples/library/page-authoring/adaptiveForm/link

Reply:
using `.../page-authoring/adaptive-form/link` for consistency with other urls
Will update later when structure is finalised